spec.md

```diff
@@ -1389,80 +1353,32 @@ $ curl 'http://username:password@service-broker-url/v2/service_instances/:instan
 
 #### Body
 
-All response bodies MUST be a valid JSON Object (`{}`). This is for future
-compatibility; it will be easier to add fields in the future if JSON is
-expected rather than to support the cases when a JSON body might or might not
-be returned.
-
-For success responses, the following fields are supported. Others will be
-ignored. For error responses, see [Service Broker
-Errors](#service-broker-errors).
+For success responses, the following fields are valid:
 
 | Response field | Type | Description |
 | --- | --- | --- |
 | operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Last Operation](#polling-last-operation) endpoint in a percent-encoded query parameter. If present, MUST be a non-empty string. |
 
-\* Fields with an asterisk are REQUIRED.
-
 ```
 {
   "operation": "task_10"
 }
 ```
 
-## Service Broker Errors
-
-### Response
-
-Service Broker failures beyond the scope of the well-defined HTTP response
-codes listed above (like `410 Gone` on [Deprovisioning](#deprovisioning)) MUST
-return an appropriate HTTP response code (chosen to accurately reflect the
-nature of the failure) and a body containing a valid JSON Object (not an
-array).
-
-#### Body
-
-All response bodies MUST be a valid JSON Object (`{}`). This is for future
-compatibility; it will be easier to add fields in the future if JSON is
-expected rather than to support the cases when a JSON body might or might not
-be returned.
-
-For error responses, the following fields are valid; while other properties
-MAY appear, Platforms MAY choose to ignore them:
-
-| Response Field | Type | Description |
-| --- | --- | --- |
-| error | string | A single word uniquely identifying the error condition. |
-| description | string | A meaningful error message explaining why the request failed. |
-
-```
-{
-  "error": "QuotaExceeded",
-  "description": "Your account has exceeded its quota for Service Instances. Please contact support at http://support.example.com."
-}
-```
-
-All errors defined in this specification include a corresponding `error` value
-that SHOULD be used in the error response message. Each error definition will
-also include a `description` property that is RECOMMENDED to be used. However,
-the Service Broker MAY use a different `description` string if appropriate, for
-example, to specify a description in a different language.
-
 ## Orphans
 
-The Platform is the source of truth for Service Instances and bindings. Service
-Brokers are expected to have successfully provisioned all the Service Instances
-and bindings that the Platform knows about, and none that it doesn't.
+The Platform is the source of truth for Service Instances and Service Bindings.
+Service Brokers are expected to have successfully provisioned all of the Service
+Instances and Service Bindings that the Platform knows about, and none that it
+doesn't.
```

Comment:
I know you just added the word "Service" in there so you didn't really change anything, but this paragraph is really kind of weird.  While there is no normative text in there so people can basically ignore it, putting requirements on one component based on what another component "knows about" just sounds really odd. Normally, this requirement would be backed up with normative text in the doc around our ops and thru people adhering to those ops we would implicitly get the correct net results. But just saying "make sure component A follow's component B's state" when its this spec's job to ensure it just reads oddly to me. No action required, just rambling... :-)